modules/db/src/main/java/org/mule/module/db/internal/domain/connection/OracleDbConnection.java

```diff
@@ -93,6 +98,33 @@ private Method getCreateArrayOfMethod(Connection delegate)
         return createArrayMethod;
     }
 
+    protected void resolveLobsForArrays(String typeName, Object[] elements) throws SQLException
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1161))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7681&issues=AWjt-QE0otIXnmsUG4vg&open=AWjt-QE0otIXnmsUG4vg)

[ISSUE_KEY]: AWjt-QE0otIXnmsUG4vg